build/yamls/base/crds.yml

```diff
@@ -52,3 +52,24 @@ spec:
     kind: Traceflow
     shortNames:
       - tf
+  validation:
```

Comment:
i think we can make it more restrictive by not preserving unknown fields.. but we could do that later..

Reply:
Added TODO in func validation